bin/hx

```diff
@@ -10,11 +10,12 @@ if [ ! -z "$1" ]; then
 	fi
 else
 	current=$(haxe -version 2>&1)
-	v=$(find "$ROOT/../versions/" -type l -printf "%f\n" | sort -r | fzf --prompt="Current: $current > ")
+	v=$(find "$ROOT/../versions/" -type l -exec basename {} \; | sort -r | fzf --prompt="Current: $current > ")
```

Comment:
Strange that this change was needed =/

Reply:
it is needed on OS X otherwise, it won't work and throw `find: -printf: unknown primary or operator`. With this change, it should now work on both linux and os x...

Reply:
Works indeed :+1: